bin/cylc-cat-log

```diff
@@ -152,8 +164,12 @@ def view_log(logpath, mode, tailer_tmpl, batchview_cmd=None, remote=False):
             cmd = tailer_tmpl % {"filename": logpath}
         # Ensure that the tail command dies if the parent process dies.
         ppid = os.getppid()
+        if remote:
+            fn = os.setpgrp
+        else:
+            fn = None
         proc = Popen(
```

Comment:
This distinction is needed to distinguish between tailing a local log (same host `cylc cat-log` runs on) and a remote log.

Reply:
If local, the subprocess is already in the same process group as the main command, and executing `os.setpgrp` will change its process group (to its own PID) - then killing the group won't kill the tail command.